rir/src/ir/Compiler.cpp

```diff
@@ -391,10 +391,7 @@ bool compileSpecialCall(Context& ctx, SEXP ast, SEXP fun, SEXP args_) {
                         cs << BC::dup() << BC::setShared();
 
                         // Now load index and target
-                        cs << (superAssign ? BC::ldvarSuper(target)
-                                           : BC::ldvar(target));
-                        if (superAssign)
-                            cs << BC::setShared();
+                        cs << BC::ldvar(target);
```

Comment:
When we later remove the `if (superAssign) return false;` above, I bet this will be forgotten?

Reply:
well, it will break fast :)